src/index.html

```diff
@@ -0,0 +1,231 @@
+    <meta name="keywords" content="pictures, device size" />
+    <link rel="shortcut icon" href="favicon.ico" />
+  </head>
+  <body>
+    <div class="page">
+      <header class="header">
+        <img
+          src="<%=require('./images/logo.svg')%>"
+          alt="Spots logo"
+          class="header__logo"
+        />
+      </header>
+      <main class="content">
+        <section class="profile page__section">
+          <div class="profile__avatar-container">
+            <button class="profile__avatar-btn">
+              <img
+                src="<%=require('./images/pencil-light.svg')%>"
+                alt="Pencil icon"
+                class="profile__pencil-icon"
+              />
+            </button>
+            <img
+              src="<%=require('./images/Avatar.svg')%>"
+              alt="Current user's avatar"
+              class="profile__avatar"
+            />
+          </div>
+          <div class="profile__column">
+            <h1 class="profile__name">Bessie Coleman</h1>
+            <p class="profile__description">Civil Aviator</p>
+            <button type="button" class="profile__edit-btn">
+              Edit Profile
+            </button>
+          </div>
+          <button type="button" class="profile__add-btn">
+            <img
+              src="<%=require('./images/plus.svg')%>"
+              alt="add button"
+              class="profile__add-btn-img"
+            />New Post
+          </button>
+        </section>
+        <section class="cards page__section">
+          <ul class="cards__list">
+            <template id="card-template">
+              <li class="card">
+                <img class="card__image" src="" alt="" />
+                <div class="card__content">
+                  <h2 class="card__title"></h2>
+                  <button type="button" class="card__like-btn"></button>
+                  <button type="button" class="card__delete-btn"></button>
+                </div>
+              </li>
+            </template>
+          </ul>
+        </section>
+      </main>
+      <footer class="footer">
+        <p class="footer__copyright">2024 &copy; Spots</p>
+      </footer>
+      <div class="modal" id="profile-edit-modal">
+        <div class="modal__container">
+          <button class="modal__close-btn"></button>
+          <h2 class="modal__title">Edit profile</h2>
+          <form class="modal__form" name="profile-form" novalidate>
+            <label for="profile-name-input" class="modal__label">
+              Name
+              <input
+                id="profile-name-input"
+                type="text"
+                name="name"
+                class="modal__input"
+                placeholder="Type your name"
+                minlength="2"
+                maxlength="40"
+                required
+              />
+              <span class="modal__error" id="profile-name-input-error"
+                >Please fill out this field.</span
```

Comment:
`                >Please fill out this field.</span
`

Please, remove the default error messages from `index.html` because they should appear only when the user types anything in the inputs